app/policies/project_policy.rb

```diff
@@ -0,0 +1,42 @@
+class ProjectPolicy < ApplicationPolicy
+  authorize :user, allow_nil: true
+
+  def index?
+    true
+  end
+
+  def new?
+    true
+  end
+
+  def create?
+    user.present?
+  end
+
+  def destroy?
+    owner?
+  end
+
+  def edit?
+    project_membership.present?
+  end
+
+  def show?
+    project_membership.present?
```

Comment:
```suggestion
    true
```
Leaning on HW: Любой может видеть список и **страницу** самого проекта.